### PR TITLE
chore(devenv): introduce skip-to-content link

### PR DESCRIPTION
**Pull request by asudoh**
### Related Ticket(s)

Refs #3897.

### Changelog

**New**

- Skip-to-content link in Web Components' Storybook environment.

### CODE REVIEW

**Comment by jeffchew** [comment]
@asudoh thank you for this change!

@kennylam @asudoh just curious if the `skip to content` link is something built into the UI Shell, or this is something specifically configured in the React storybook as well?

**Comment by kennylam** [comment]
> @kennylam @asudoh just curious if the `skip to content` link is something built into the UI Shell, or this is something specifically configured in the React storybook as well?

@jeffchew Yes, `skip to content` is already included in the React UI shell's `HeaderNav` component.

**Comment by asudoh** [comment]
@jeffchew `carbon-components-react` has the corresponding component, but `carbon-web-components` doesn't, because skip-to-content link essentially is not shell and I was personally hesitant to implement it as a UI shell component. But if we want it in UI shell I can go ahead with it. Let me know.

**Comment by jeffchew** [comment]
> @jeffchew `carbon-components-react` has the corresponding component, but `carbon-web-components` doesn't, because skip-to-content link essentially is not shell and I was personally hesitant to implement it as a UI shell component. But if we want it in UI shell I can go ahead with it. Let me know.

I don't have a strong opinion on this, any pros and cons either way?

cc: @kennylam

**Comment by asudoh** [comment]
The pro is that less confusion on the responsibility of skip-to-content link, where it's essentially in application. The con is potentially some people cannot figure out how, but I think it's less likely.

**Comment by jeffchew** [comment]
> The pro is that less confusion on the responsibility of skip-to-content link, where it's essentially in application. The con is potentially some people cannot figure out how, but I think it's less likely.

So the idea is that we leave it up to the adopters to apply it themselves, rather than have it be part of something like UI Shell. 

Want to get designer/FA opinion on this: @wonilsuhibm @larahanlon2 @oliviaflory @JennySanchez @jacobottesen-dgc @photodow

**Comment by jeffchew** [comment]
@asudoh and others, one other thought is that we can offer `skip-to-content` as a separate component entirely?

**Comment by photodow** [comment]
@jeffchew @asudoh for me I'm curious about how important is this to be applied consistently. If we leave it up to adopters it's most likely not going to get applied. If we're ok with, and disabled users don't really need it then I'm ok with introducing it separately. The way it's implemented today doesn't look great. It might also be worth bringing up to our accessibility team, and see how much value it is and how necessary it is. To me including it in the header provides users one less thing to think about when it comes to accessibility, and it's small on our part to implement.

**Comment by asudoh** [comment]
@jeffchew @photodow Thank you for your thoughts/feedbacks. Given them I think defining the component as a non-UI-shell component makes the most sense.

**Comment by asudoh** [comment]
Updated so we ship the skip-to-link as a component.

File: packages/web-components/src/components/skip-to-content/__stories__/README.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/skip-to-content/skip-to-content';
+```
+
+### HTML
+
+```html
+<dds-skip-to-content></dds-skip-to-content>
```
Comment (jeffchew) [inline packages/web-components/src/components/skip-to-content/__stories__/README.stories.mdx:25]: Thank you @asudoh ! For the README, can you add documentation on the `href` property?

Comment (asudoh) [reply packages/web-components/src/components/skip-to-content/__stories__/README.stories.mdx:25]: Thank you for the feedback @jeffchew! After thinking a while; Now trying to implement this in `carbon-web-components`: carbon-design-system/carbon-web-components#486 - The doc includes the prop description. Will update this PR once carbon-design-system/carbon-web-components#486 is merged and released.

Comment (asudoh) [reply packages/web-components/src/components/skip-to-content/__stories__/README.stories.mdx]: Made an update to use `<bx-skip-to-content>` from `carbon-web-components`.

**Review by jeffchew — APPROVED** [review]
LGTM! Just need to fix merge conflicts then this should be good to go.

**Comment by asudoh** [comment]
Thanks @jeffchew - Solved the merge conflict.

**Pull request merged**